src/xdocs/google_style.xml

```diff
@@ -2432,6 +2432,23 @@
           </table>
         </div>
       </subsection>
+      <subsection name="Severity Level" id="Google_Severity_Level">
+        <p>
+          To adjust the default severity level for violations using Checkstyle's
+          Google style configuration, set the following system property:
+          <br/>
+          <br/>
+          <code>org.checkstyle.google.severity</code>
```

Comment:
replace the 2 `<br/>` with `</p><p>`. Your basically trying to create a paragraph separation without using it.

Reply:
done 

https://github.com/checkstyle/checkstyle/blob/012932d2daae8b793e2fc2330dbde92b0b15f751/src/xdocs/google_style.xml#L2438-L2441